addons/explosives/stringtable.xml

```diff
@@ -529,5 +530,103 @@
             <English></English>
             <Polish>Moduł ten pozwala dostosować opcje związane z ładunkami wybuchowymi, ich podkładaniem oraz rozbrajaniem.</Polish>
         </Key>
+=======
+        <Key ID="STR_ACE_Explosives_DaisyChainName">
+            <English>Daisy Chain</English>
+            <German>Daisy Chain</German>
```

Comment:
@LorenLuke, just leave the english entry here and remove the rest. That way it's easier for traslators to find the missing entries to work on.